github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -37,19 +37,17 @@ presubmits:
               topologyKey: kubernetes.io/hostname
       containers:
       - image: quay.io/kubevirtci/golang:v20210316-d295087
+        env:
+        - name: "GIMME_GO_VERSION"
+          value: "1.13.8"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - >
-            apt update &&
-            apt install gettext-base -y &&
-            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S &&
-            tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/ &&
-            export PATH=/usr/local/go/bin:$PATH &&
+            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM &&
             export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0 &&
```

Comment:
Can you pls use `env` entries and remove the two exports?

Reply:
Done